### DISCUSSION TITLE
Community additions to BMI

### DISCUSSION THREAD

**Discussion by mdpiper** [csdms/bmi#117] [General]
BMI specs and examples for [Javascript](https://github.com/uihilab/BMI-JS) and [Julia](https://github.com/Deltares/BasicModelInterface.jl) have been created by researchers in the community. I'd like to add these to the [docs](https://bmi.readthedocs.io/en/stable/). I'll create a PR soon, but I want to give everyone a heads up in case there's any discussion.

**Comment by mdpiper** [comment]
I'm thinking about adding these to a new table, titled something like _Table 2: Community-contributed BMI language specifications._

**Comment by alee** [comment]
(from BMI Council meeting) Niels Drost mentioned it would be good to make the table more explicitly labelled as _Community Contributed_ since CSDMS isn't officially supporting them

**Comment by mdpiper** [comment]
Here's the PR: #120. I added a new table to the README and the docs.

**Comment by alee** [comment]
also might be worth considering what it would take to graduate a community contributed BMI to an officially supported BMI through some incubation type process (at some point, not part of #120)
